apps/akasha/src/components/feed-page/feed-page.tsx

```diff
@@ -11,7 +11,7 @@ import { getFeedCustomEntities } from './feed-page-custom-entities';
 import { redirectToPost } from '../../services/routing-service';
 import EntryCardRenderer from './entry-card-renderer';
 import routes, { POST } from '../../routes';
-import { application as loginWidget } from '@akashaproject/ui-widget-login/lib/bootstrap';
+import { application as loginWidget } from '@akashaproject/ui-widget-login/lib/index';
```

Comment:
this can be also shortened to `import { application as loginWidget } from '@akashaproject/ui-widget-login/lib'`